### ISSUE TITLE
Question: Replace libdbusmenu-glib with unity7/libdbusmenu-glib-ubuntu?

### ISSUE THREAD

**Issue by nobicycle** [manjaro/unity-for-manjaro#4]
Hello ad thanks for Uity7.

During install:

`Replace libdbusmenu-glib with unity7/libdbusmenu-glib-ubuntu? `

Dependencies are: ibdbusmenu-glib, xfce4-panel ...

Will xfce4-panel work OK with unity7/libdbusmenu-glib-ubuntu?

Best wishes

**Comment by RudraSwat** [comment]
`libdbusmenu-glib` is identical to `libdbusmenu-glib-ubuntu`, so it should work fine with xfce4-panel. We're dropping `libdbusmenu-glib-ubuntu` however, so you'll want to reinstall `libdbusmenu-glib` later.

**Comment by FabioLolix** [comment]
The issue was that `libdbusmenu-glib-ubuntu` had `replaces=(libdbusmenu-glib)` which is to be used when a package is being deprecated, in this case `libdbusmenu-glib` is not deprecated and replaces should not have been added in first place

**Comment by RudraSwat** [comment]
Yep, which is exactly why we dropped that package (I hadn't added that package; it came from chenxiaolong's Unity-for-Arch port)

**Comment by FabioLolix** [comment]
> which is exactly why we dropped that package

Could simply remove `replaces=(libdbusmenu-glib)`, anyway if it works using less custom packages is better
